### DISCUSSION TITLE
项目容器化

### DISCUSSION THREAD

**Discussion by sunist-c** [Arondight/Adachi-BOT#917] [Ideas]
最近迁移了下服务器，也想把bot交给prometheus监控一下，遂想把这玩意容器化，彻底解决chrome的问题和各种奇怪的安装问题，不知道主分支这边有没有这方面的想法，发个discussion询问一下

**Comment by Arondight** [comment]
1. 没有容器化意向
2. https://github.com/htoila/Adachi-BOT

**Reply by sunist-c** [reply]
ok，感谢！
